src/dfx/assets/new_project_node_files/webpack.config.js

```diff
@@ -1,41 +1,10 @@
+require("dotenv").config();
 const path = require("path");
 const webpack = require("webpack");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const TerserPlugin = require("terser-webpack-plugin");
 const CopyPlugin = require("copy-webpack-plugin");
 
-function initCanisterEnv() {
-  let localCanisters, prodCanisters;
-  try {
-    localCanisters = require(path.resolve(
-      ".dfx",
-      "local",
-      "canister_ids.json"
-    ));
-  } catch (error) {
-    console.log("No local canister_ids.json found. Continuing production");
-  }
-  try {
-    prodCanisters = require(path.resolve("canister_ids.json"));
-  } catch (error) {
-    console.log("No production canister_ids.json found. Continuing with local");
-  }
-
-  const network =
-    process.env.DFX_NETWORK ||
-    (process.env.NODE_ENV === "production" ? "ic" : "local");
-
-  const canisterConfig = network === "local" ? localCanisters : prodCanisters;
-
-  return Object.entries(canisterConfig).reduce((prev, current) => {
-    const [canisterName, canisterDetails] = current;
-    prev[canisterName.toUpperCase() + "_CANISTER_ID"] =
-      canisterDetails[network];
-    return prev;
-  }, {});
-}
-const canisterEnvVariables = initCanisterEnv();
```

Comment:
Do any of these changes need to be applied to existing projects?

Reply:
They don't need to be. Existing projects only need to worry about the change to the generated file, which I noted in the migration notes. They don't need to remove this block and switch to dotenv if they don't want to